<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1063</width>
    <height>830</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>20</y>
     <width>581</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Analytics Dashboard</string>
   </property>
  </widget>
  <widget class="QComboBox" name="yearFilter">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>130</y>
     <width>151</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="monthFilter">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>130</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="display_graph_widget" native="true">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>170</y>
     <width>1011</width>
     <height>601</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="chartFilter">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>70</y>
     <width>451</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
